test/e2e/framework/kubelet/stats.go

```diff
@@ -640,14 +640,14 @@ func (r *ResourceMonitor) GetCPUSummary() NodesCPUSummary {
 	return result
 }
 
-// GetMasterNodeCPUSummary returns summary of master node CPUs.
-func (r *ResourceMonitor) GetMasterNodeCPUSummary(summaryPerNode NodesCPUSummary) NodesCPUSummary {
+// GetControlPlaneCPUSummary returns summary of control plane CPUs.
+func (r *ResourceMonitor) GetControlPlaneCPUSummary(summaryPerNode NodesCPUSummary) NodesCPUSummary {
 	result := make(NodesCPUSummary)
-	var masterSummary ContainersCPUSummary
+	var controlPlaneSummary ContainersCPUSummary
 	var nodesSummaries []ContainersCPUSummary
 	for node, summary := range summaryPerNode {
 		if strings.HasSuffix(node, "master") {
```

Comment:
It seems we're not actually changing the node name in this PR? We will need to do that, won't we?